server/controller/admin_controller.ts

```diff
@@ -58,3 +59,21 @@ export const handleAdminRestoreUser = async (
 		next(err);
 	}
 };
+
+export const handleAdminGetPosts = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+) => {
+	try {
+		const index = parseInt(req.query.index as string) - 1 || 0;
+		const perPage = parseInt(req.query.perPage as string) || 10;
```

Comment:
@Siltarae `nit` 
매직 넘버는 상수화 하는 것이 좋습니다.
`handleGetUsers`, `handleAdminGetPosts` 함수에서 `index`, `perPage` 기본값 설정이 반복되고 있는데요, 이를 상수로 정의하면 가독성/유지보수 측면에서 좋습니다.(추가로 음수 방어로직도 추가되면 좋을 것 같네요)
```ts 
const DEFAULT_INDEX = 0;
const DEFAULT_PER_PAGE = 10;
```

```suggestion
		let index = parseInt(req.query.index as string) - 1 || DEFAULT_INDEX;
		let perPage = parseInt(req.query.perPage as string) || DEFAULT_PER_PAGE;

		if (index < 0 || perPage < 0) {
			index = DEFAULT_INDEX;
			perPage = DEFAULT_PER_PAGE;
		}
```